src/app/lib/checkbox/checkbox.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="hc-checkbox-container" [ngClass]="{'disabled': disabled}">
+    <input type="checkbox" [attr.id]="name" [disabled]="disabled" [(ngModel)]="model" />
+    <label class="hc-checkbox-overlay" [attr.for]="name" [ngClass]="color"></label>
```

Comment:
Lint caught this one error.  We got rid of the color option but forgot to remove it from the html.

Reply:
Running `ng lint` locally didn't catch this.  I'll have to investigate why.